src/test/java/com/cloudbees/jenkins/support/util/WordReplacerTest.java

```diff
@@ -153,6 +153,20 @@ public void caseReplacementsTest() {
         Assert.assertEquals("1 1 b,B.c:C abc ABC ignored", inputSB.toString());
     }
 
+    @Test
+    public void indexOutOfBoundsExceptionTest() {
+        String input = "input one input";
+        String[] words = new String[]{   "input",   "one"};
+        String[] replaces = new String[]{"", ""};
```

Comment:
How is this situation reached? I mean when would we want to replace by an empty string?

Reply:
As the class WordReplacer is a sort of utility class, it should work in any case, as this one. Specifically, in the use case of this plugin this test is a concrete case of a replacement by a word with less length, which is a valid use case.